app/templates/repositories/show/dois/new.hbs

```diff
@@ -25,6 +25,7 @@
         <DoiSubjects @model={{model.doi}} @form={{form}} />
         <DoiLanguage @model={{model.doi}} @form={{form}} />
         <DoiContributors @model={{model.doi}} @form={{form}} />
+        <DoiAlternateIdentifiers @model={{model.doi}} @form={{form}} />
```

Comment:
Have you considered just naming this Identifiers to match the model? I know in terms of the XML metadata it is alternate metadata but the model is saying identifiers.

Reply:
I think the advantage of using this name is that one knows which component to modify when something related to alternate id needs modification. Thats the label on the UI and the metadata schema. It is unfortunate that the jsonapi doesn't follow that convention